src/components/shared/mediaText.js

```diff
@@ -30,9 +30,36 @@ function MediaText (props) {
     
     let mediaCol = "col-xs-12";
     let textCol = "col-xs-12";
+    let textColBg;
+    let textColPadding;
     let wrapperCol;
     let headingClass;
+    let headingColor;
     let textOrButtons = mediaDescription || mediaButtons ? true : false;
+    
+    switch(mediaBgColor) {
+        case "Blue Muted":
+            textColBg = " uog-blue-muted";
```

Comment:
I feel like someday a person could easily forget to include that first space. Could we use the [classnames](https://www.npmjs.com/package/classnames) package in the rendering of the classnames so we don't have to worry about spaces?

Reply:
Done